### ISSUE TITLE
Should we have read me for all stores ?

### ISSUE THREAD

**Issue by kunjee17** [Dzoukr/CosmoStore#34]
@Dzoukr  I was thinking that we should have read me for all store. How to set up things and all. Not detailed as current one but simpler one? And we can have github pages as well.

**Comment by Dzoukr** [comment]
Yeah... I think we can do it in a same way as it is now in https://github.com/Dzoukr/Feliz.Bulma repo where there is folder for `docs` (using GH pages). Maybe using Fable?

**Comment by kunjee17** [comment]
I am practically OK with any thing. Even simple read me files in respective folders.

**Comment by Dzoukr** [comment]
Ok, maybe this would be even easier
